src/mixin/java/com/gtnewhorizons/angelica/mixins/early/angelica/archaic/MixinSplashProgress.java

```diff
@@ -36,6 +37,8 @@ public class MixinSplashProgress {
     private static float memoryColorPercent;
     private static long memoryColorChangeTime;
     private static FontRenderer fontRenderer = null;
+    @Unique
+    private static Boolean angelica$isArchaicFixPresent = null;
```

Comment:
because I also need the state where it hasn't been set yet. I detect that this boolean hasn't been set yet by checking if it's null. That way the check if archaicfix is present only needs to be done once. Alternatively a second boolean could be introduced where I keep track on if the first one has been properly set but this is neater imo